core/src/Network/GRPC/LowLevel/GRPC/MetadataMap.hs

```diff
@@ -29,8 +32,9 @@ import qualified Data.Map.Strict as M
    Just "y"
 -}
 
-newtype MetadataMap = MetadataMap {unMap :: M.Map ByteString [ByteString]}
-  deriving Eq
+newtype MetadataMap = MetadataMap 
+  {unMap :: M.Map ByteString [ByteString]}
+  deriving (Data, Eq, Ord, Typeable)
```

Comment:
It is not

Reply:
From the `Data.Typeable` documentation:

```
Since GHC 7.10, all types automatically have Typeable instances derived. This is in contrast to previous releases where Typeable had to be explicitly derived using the DeriveDataTypeable language extension.
```

Reply:
til